libbeat/template/processor.go

```diff
@@ -0,0 +1,259 @@
+package template
```

Comment:
Did this file change in any way? Seems Github / Git did not catch up on the moving :-(